### PR TITLE
Remove width & height from get_visible_nodes API.

### PR DESCRIPTION
**Pull request by SirVer**

### CODE REVIEW

**Comment by SirVer** [comment]
Depends on #141.

File: src/octree/mod.rs [hunk 1]
```diff
@@ -73,10 +71,7 @@ fn size_in_pixels(
     ] {
         rv = rv.grow(project(matrix, p));
     }
-    Vector2::new(
```
Comment (thomasschiwietz) [inline src/octree/mod.rs]: Did you think about the case if the aspect ratio (width / height) is not close to 1.0? I think the new approach is only correct if the aspect ratio is close to 1.0. How about replacing the width, height parameters by an aspect ratio parameter and adjust the normalized width or height accordingly?

Comment (SirVer) [reply src/octree/mod.rs]: I thought about this, but felt that it was not adding a ton to the heuristic - I mean at worst the aspect ration will be like 2/3 or so in which case this should still be a reasonable approximation. Do you think it makes sense?

Comment (thomasschiwietz) [reply src/octree/mod.rs]: Ok. Could put a comment in the code about it?

Comment (SirVer) [reply src/octree/mod.rs]: done.

**Comment by wally-the-cartographer** [comment]
Merge requested by authorized user SirVer. Merge queue now has a length of 1.

**Pull request merged**